package.json

```
@@ -1,14 +1,16 @@
 {
   "name": "slow-json-stringify",
-  "version": "2.0.1",
-  "description": "The slowest JSON stringifier in the galaxy (:",
+  "version": "2.1.0",
+  "description": "The slow.. well actually fastest JSON stringifier in the galaxy.",
   "source": "src/sjs.mjs",
   "main": "dist/sjs.js",
   "module": "dist/sjs.mjs",
+  "exports": "./dist/sjs.modern.js",
+  "umd:main": "dist/sjs.umd.js",
   "unpkg": "dist/sjs.umd.js",
   "types": "index.d.ts",
   "scripts": {
-    "build": "microbundle",
+    "build": "microbundle --compress --strict",
```

These are both the default values.

```suggestion
    "build": "microbundle",
```